docs/standard/threading/timers.md

```diff
@@ -14,19 +15,21 @@ author: "rpetrusha"
 ms.author: "ronpet"
 ---
 # Timers
-Timers are lightweight objects that enable you to specify a delegate to be called at a specified time. A thread in the thread pool performs the wait operation.  
+
+The <xref:System.Threading.Timer?displayProperty=nameWithType> class enables you to continuously call a delegate at specified time intervals. You also can use this class to schedule the single call to a delegate in a specified time interval. The delegate is executed each time on a <xref:System.Threading.ThreadPool> thread.
```

Comment:
the single call --> a single call
executed each time --> executed

Reply:
The first deleted paragraph is still important, since it provides background for using the control that doesn't have to be gleaned from the example. I'd include a modified version:


You create a \<xref:System.Threading.Timer?displayProperty=nameWithType> object by passing one of its constructors a \<xref:System.Threading.TimerCallback> delegate that defines the callback method, an optional state object that is passed to the callback, an initial raise time, and a time representing the period between callback invocations. To cancel a pending timer, call the \<xref:System.Threading.Timer.Dispose%2A?displayProperty=nameWithType> method.   
-